standards/SPIFFE_Federation.md

```diff
@@ -243,16 +245,16 @@ Each SPIFFE Federation relationship is configured with the following parameters
 
 It is important that these three parameters are configured explicitly, the values cannot be securely inferred from each other.
 
-For example, one might be tempted to infer the SPIFFE trust domain name from the host portion of the Endpoint URL. Firstly, we emphasize that a SPIFFE trust domain name need not be a registered DNS name.  Secondly, inferring this as a default is dangerous because it would allow anyone who can get a file served from a particular DNS name to assert trust roots for the SPIFFE trust domain of the same name. 
+For example, one might be tempted to infer the SPIFFE trust domain name from the host portion of the Endpoint URL. Inferring this as a default is dangerous because it could allow anyone who can get a file served from a particular DNS name to assert trust roots for the SPIFFE trust domain of the same name. 
```

Comment:
s/. Inferring this as a default/; inferring a relationship/ ?

Reply:
From SIG-Spec call: This is dangerous because ...